chat-plugins/economy.js

```diff
@@ -64,98 +64,98 @@ exports.commands = {
-				Economy.logTransaction(user.name + " has taken " + amount + ((amount === 1) ? " " + currencyName + " " : " " + currenyPlural + " ") + " from " + targetUser + ". (Reason: " + reason + ") They now have " + newAmount + (newAmount === 1 ? " " + currencyName + "." : " " + currenyPlural + "."));
+				this.sendReply("You removed " + amount + ((amount === 1) ? " " + currencyName + " " : " " + currencyPlural + " ") + " from " + Chat.escapeHTML(targetUser));
+				Economy.logTransaction(user.name + " has taken " + amount + ((amount === 1) ? " " + currencyName + " " : " " + currencyPlural + " ") + " from " + targetUser + ". (Reason: " + reason + ") They now have " + newAmount + (newAmount === 1 ? " " + currencyName + "." : " " + currencyPlural + "."));
 			});
 		});
 	},
 
-	confirmtransfercurrency: 'transfercurrency',
-	transfercurrency: function (target, room, user, connection, cmd) {
-		if (!target) return this.sendReply("Usage: /transfercurrency [user], [amount]");
+	confirmtransferstardust: 'transferstardust',
+	transferstardust: function (target, room, user, connection, cmd) {
+		if (!target) return this.sendReply("Usage: /transferstardust [user], [amount]");
 		let splitTarget = target.split(',');
 		for (let u in splitTarget) splitTarget[u] = splitTarget[u].trim();
-		if (!splitTarget[1]) return this.sendReply("Usage: /transfercurrency [user], [amount]");
+		if (!splitTarget[1]) return this.sendReply("Usage: /transferstardust [user], [amount]");
 
 		let targetUser = (Users.getExact(splitTarget[0]) ? Users.getExact(splitTarget[0]).name : splitTarget[0]);
-		if (toId(targetUser).length < 1) return this.sendReply("/transfercurrency - [user] may not be blank.");
-		if (toId(targetUser).length > 18) return this.sendReply("/transfercurrency - [user] can't be longer than 18 characters.");
+		if (toId(targetUser).length < 1) return this.sendReply("/transferstardust - [user] may not be blank.");
+		if (toId(targetUser).length > 18) return this.sendReply("/transferstardust - [user] can't be longer than 18 characters.");
 
 		let amount = Math.round(Number(splitTarget[1]));
-		if (isNaN(amount)) return this.sendReply("/transfercurrency - [amount] must be a number.");
-		if (amount > 1000) return this.sendReply("/transfercurrency - You can't transfer more than 1000 currency at a time.");
-		if (amount < 1) return this.sendReply("/transfercurrency - You can't transfer less than one buck.");
+		if (isNaN(amount)) return this.sendReply("/transferstardust - [amount] must be a number.");
+		if (amount > 1000) return this.sendReply("/transferstardust - You can't transfer more than 1000 Stardust at a time.");
+		if (amount < 1) return this.sendReply("/transferstardust - You can't transfer less than one Stardust.");
 
 		Economy.readMoney(user.userid, money => {
-			if (money < amount) return this.sendReply("/transfercurrency - You can't transfer more currency than you have.");
-			if (cmd !== 'confirmtransfercurrency') {
```

Comment:
If we allow `transfercurrency` and `transferstardust` we need to set this up as:
```javascript
if (cmd !== 'confirmtransfercurrency' && cmd !== 'confirmtransferstardust') {
```
So that they both work properly